test/Test/Pos/Types/CoinSpec.hs

```diff
@@ -135,9 +132,6 @@ overflowIntegerCausesError =
 coinToIntegralToCoin :: C.Coin -> Property
 coinToIntegralToCoin = C.mkCoin . C.unsafeGetCoin .=. identity
 
-wordToCoinToWord :: Word64 -> Property
```

Comment:
This test isn't all too important but it's useful to know turning a `Word64` into a coin and then using `unsafeGetCoin` doesn't change anything.